client/src/pages/charactersGalery/ChracterGalery.module.css

```diff
@@ -0,0 +1,33 @@
+.footer {
+    display: flex;
+    font-size: 1.5rem;
+    color: white;
+    justify-content: center;
+}
+
+ul {
+    display: flex;
+    gap: 0.5em;
+    gap: 0.5rem;
+    justify-content: center;
+    list-style-type: none;
+}
+
+il {
+    background: aqua;
```

Comment:
Best practice is to defined all your color in the theme instead for more flexibility